bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -59,21 +59,6 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
-                ->arrayNode('error_handling')
```

Comment:
We cannot remove this config node, because this would be a breaking change. 
We need to keep it, even if it has no use anymore and deprecate it, so that we can remove it with Pimcore 11.  Use  `->setDeprecated('...')` for that.

Reply:
@Yivan Sorry, I should have commented this info onto the actual code line from the beginning on 😊 I hope it's now clear why I mean. Thanks!

Reply:
@brusch Thanks for the info! This is done, I updated the PR.